packages/twenty-front/src/hooks/__tests__/useCombinedRefs.test.ts

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { renderHook } from '@testing-library/react';
+
+import { useCombinedRefs } from '~/hooks/useCombinedRefs';
+
+describe('useCombinedRefs', () => {
```

Comment:
@charlesBochet There's lots of "complex" hooks. I suppose we're writing tests for them too. I suggest doing it in smaller PRs to make reviews much easier.

Reply:
Let's track the whole work within #2992. We can discuss on this ticket what next hooks should be tackled